100=ContentProvider.__init__()
101=ContentProvider: %s Chargement termin\u00e9

110=ContentProvider.registerInstance()
111=ContentProvider: enregistre l'instance d'un ParameterizedContentProvider avec l'argument: %s


200=ParameterizedProvider.__init__()
201=ParameterizedProvider%s Chargement termin\u00e9

210=ParameterizedProvider.createContentIdentifier()
211=Identifier: %s -> %s

220=ParameterizedProvider.queryContent()
221=Identitifer: %s ... Termin\u00e9
222=IllegalIdentifierException: %s
223=Exception python inattendue avec le traceback: %s

230=ParameterizedProvider.compareContentIds()
231=Identifiers: %s - %s ... semble \u00eatre le m\u00eame
232=Identifiers: %s - %s ... ne semble pas \u00eatre le m\u00eame


300=DataSource.__init__()
301=DataSource: Chargement termin\u00e9

310=DataSource._getDataSource()
311=Essaie d'obtenir la source de données avec l'URL: %s
312=A obtenu la source de données enregistrée sous: %s
313=A obtenu la source de données de l'URL: %s
314=Essaie de créer une nouvelle source de données avec l'URL: %s
315=Une nouvelle source de données a été créée avec l'URL: %s
316=La nouvelle source de données avec l'URL: %s a été enregistrée sous: %s

320=DataSource._getUser()
321=Erreur: Impossible de charger de contenu avec un Identifier invalide: %s
322=Erreur: Impossible de r\u00e9cup\u00e9rer l'utilisateur à partir de l'Identifier: %s

330=DataSource._getUserName()
331=Erreur: Impossible de charger de contenu sans Utilisateur: %s

340=DataSource.queryClosing()
341=La base de donn\u00e9e: %s a \u00e9t\u00e9 ferm\u00e9e


400=DataBase.__init__()
401=DataBase: Chargement termin\u00e9

410=DataBase.createDataBase()
411=Essaie de créer la base de données avec HsqlDB version: %s
412=Création réussie de la base de données
413=Impossible de créer la base de données : la version de HsqlDB installée: %s est inférieure à la version minimale prise en charge: %s


500=ContentUser.__init__()
501=Le Service: %s n'est pas disponible... Veuillez installer l'extension: OAuth2OOo!!!
502=Impossible de retrouver l'utilisateur: %s chez le fournisseur: le r\u00e9seau est hors ligne!!!
503=L'utilisateur: %s est inconnu chez ce fournisseur
504=Impossible d'inserer l'utilisateur: %s dans la base de donn\u00e9es
505=ContentUser chargement termin\u00e9

510=ContentUser.getDocumentContent()
511=Erreur: %s - %s


600=Content.__init__()
601=Content chargement termin\u00e9

610=Content._getMetaData()
611=Erreur: Impossible de charger le contenu avec un Identifier incomplet: %s
612=Erreur: Impossible de r\u00e9cup\u00e9rer les m\u00e9tadonn\u00e9es pour l'ID: %s et l'Uri: %s

620=Content._updateFolderContent()
621=Le contenu du dossier: %s est obtenu avec Request!!!

